commit-event-service/src/test/scala/io/renku/commiteventservice/events/categories/common/CommitInfoFinderSpec.scala

```diff
@@ -52,89 +57,70 @@ class CommitInfoFinderSpec
   "findCommitInfo" should {
 
     "fetch commit info from the configured url " +
-      "and return CommitInfo if OK returned with valid body" in new TestCase {
-
-        stubFor {
-          get(s"/api/v4/projects/$projectId/repository/commits/$commitId")
-            .withAccessToken(maybeAccessToken)
-            .willReturn(okJson(responseJson.toString()))
-        }
-
-        finder.findCommitInfo(projectId, commitId).unsafeRunSync() shouldBe CommitInfo(
+      "and return CommitInfo if OK returned with valid body - case with Personal Access Token" in new TestCase {
```

Comment:
Could we maybe do the same as I mentioned in the previous comment and remove the test for oauth token case